pkg/kfapp/kustomize/kustomize.go

```diff
@@ -216,7 +216,8 @@ func (kustomize *kustomize) Dump(resources kftypesv3.ResourceEnum) error {
 		if err != nil {
 			return err
 		}
-		fmt.Println(string(data))
+		fmt.Print(string(data))
```

Comment:
Add `---` is okay, but I do want to keep printing original data with new line.

Reply:
Okay, I've updated the PR :).